csrc/tensor_metadata.h

```diff
@@ -115,4 +115,9 @@ inferAndValidateAllocationSizesAndStrides(
     TensorView* tv,
     ExpressionEvaluator ee);
 
+// Given a tensorview, compute the strides according to the allocation domain
+// for re-striding the corresponding ATen tensor.
+std::vector<int64_t> computeStrides(
```

Comment:
Nit: I smell a YAGNI -- I think it's premature to expose this via a header whereas the only use is in tensor_metadata.cpp. I'd instead leave this helper function in tensor_metadata.cpp.

Reply:
I am using it in `nodes.cpp`.

Reply:
Sorry for missing the fact that computeStrides is used in nodes.cpp. However, the same thing -- it's only used in one source location so I'd rather it be defined in the same file (i.e. nodes.cpp). (It **could** be reused elsewhere but YAGNI)